tap_twilio/streams.py

```diff
@@ -12,313 +12,125 @@
-                'path': 'Accounts/{ParentId}/Transcriptions.json',
-                'data_key': 'transcriptions',
-                'key_properties': ['sid'],
-                'replication_method': 'INCREMENTAL',  # Fetch ALL, filter results
-                'replication_keys': ['date_updated'],
-                'params': {},
-                'pagingation': 'root'
-            },
-            # pylint: disable=line-too-long
-            # Reference: https://www.twilio.com/docs/voice/api/queue-resource#read-multiple-queue-resources
-            'queues': {
-                'api_url': 'https://api.twilio.com',
-                'api_version': '2010-04-01',
-                'path': 'Accounts/{ParentId}/Queues.json',
-                'data_key': 'queues',
-                'key_properties': ['sid'],
-                'replication_method': 'INCREMENTAL',  # Fetch ALL, filter results
+                'replication_method': 'FULL_TABLE',
                 'replication_keys': ['date_updated'],
                 'params': {},
-                'pagingation': 'root'
-            },
-            # pylint: disable=line-too-long
-            # Reference: https://www.twilio.com/docs/sms/api/message-resource#read-multiple-message-resources
-            'messages': {
-                'api_url': 'https://api.twilio.com',
-                'api_version': '2010-04-01',
-                'path': 'Accounts/{ParentId}/Messages.json',
-                'data_key': 'messages',
-                'key_properties': ['sid'],
-                'replication_method': 'INCREMENTAL',  # Filter query
-                'replication_keys': ['date_sent'],
-                'bookmark_query_field_from': 'DateSent>',  # Daily
-                'bookmark_query_field_to': 'DateSent<',
-                'params': {},
-                'pagingation': 'root',
+                'pagination': 'root',
                 'children': {
-                    # pylint: disable=line-too-long
-                    # Reference: https://www.twilio.com/docs/sms/api/media-resource#read-multiple-media-resources
-                    'message_media': {
-                        'api_url': 'https://api.twilio.com',
-                        'api_version': '2010-04-01',
-                        'path': 'Accounts/{AccountSid}/Messages/{ParentId}/Media.json',
-                        'data_key': 'media_list',
-                        'key_properties': ['sid'],
-                        'replication_method': 'FULL_TABLE',  # ALL for parent Address
+                    'channels': {
+                        'api_url': 'https://taskrouter.twilio.com',
+                        'api_version': 'v1',
+                        'path': 'Workspaces/{ParentId}/Workers/{ParentId}/Channels',
+                        'data_key': 'channels',
+                        'key_properties': ["sid"],
+                        'replication_method': 'FULL_TABLE',
+                        'replication_keys': ['date_updated'],
```

Comment:
Same as above. `FULL_TABLE` is not compatible with the `'replication_keys': ['date_updated'],`, and it should be `INCREMENTAL` if the API endpoint is supporting it.